<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>310</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>280</width>
    <height>310</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="labelName">
     <property name="text">
      <string>Nazwa</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="Name"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelEqn">
     <property name="text">
      <string>RÃ³wnanie</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="Equation"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelType">
     <property name="text">
      <string>Jednostka</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="Type"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="labelMin">
     <property name="text">
      <string>Minimum</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QDoubleSpinBox" name="Min"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="labelMax">
     <property name="text">
      <string>Maksimum</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QDoubleSpinBox" name="Max"/>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="labelStyle">
     <property name="text">
      <string>Styl</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="StyleLCD">
       <property name="text">
        <string>LCD</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="StyleBar">
       <property name="text">
        <string>Pasek</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="1">
    <widget class="QCheckBox" name="Enabled">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Aktywny</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>270</x>
     <y>291</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>270</x>
     <y>291</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>onDialogSave()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>243</x>
     <y>289</y>
    </hint>
    <hint type="destinationlabel">
     <x>203</x>
     <y>369</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onDialogSave()</slot>
 </slots>
</ui>
